### PR TITLE
Few enhancements in main file

### PR DESCRIPTION
**Pull request by cf-sujal**
Got to know about your package after it was mentioned [here](https://docs.github.com/en/search-github/github-code-search/about-github-code-search#limitations). Found your tool interesting, and to my further bliss, got to know it was written in Go, which I am kind of familiar with. Got through the code, and tried to clean up a few things.

I have added the CHANGELOG file, thinking it could be of some help to the project. Have a Good day, Guys!

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -307,11 +310,11 @@ func printFileAnalysis(file string, limit int64, isJSON bool) error {
 
 	fmt.Printf(
 		`%s: %d lines (%d sloc)
-  type:      %s
-  mime_type: %s
-  language:  %s
-  vendored:  %t
-`,
+				type:      %s
```
Comment (lafriks) [inline main.go:313]: this should not be changed as it changes output

**Review by lafriks — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (cf-sujal) [reply main.go]: Undid the changes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# ChangeLog
+
+### [2023-03-03](https://github.com/go-enry/enry/pull/12)
```
Comment (bzz) [inline CHANGELOG.md:3]: I would suggest not to host a copy of https://github.com/go-enry/enry/pull here, as it adds maintenance burden without much value.

IMO changelog would make some sense if it records the release history, but again only if the advantages overweight the burden on maintainers.

Comment (cf-sujal) [reply CHANGELOG.md:3]: Makes sense. You guys have been maintaining the repo for a while now. What do you suggest? If it's worth adding, then let's go with it, if not, it's not required. Also, if there is already some sort of note that keeps track of things on every version bump, then maintaining CHANGELOG could be redundant.

Comment (lafriks) [reply CHANGELOG.md:3]: I also agree on not adding changelog. I don't see much value here as most value anyway are in library not in tool itself.

Comment (cf-sujal) [reply CHANGELOG.md]: Copy that. I have now removed the CHANGELOG file.

**Review by lafriks — APPROVED** [review]

**Pull request merged**